extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -125,6 +126,38 @@ export const LIFECYCLE_STAGES = {
 };
 
 
+/**
+ * Utility function that ensures any error thrown is handled by optional
```

Comment:
Can we be clear on whether the caller of a function wrapped with this can use its return value? If so, there should be explicit `return`s for the case where there's no `opt_onError` or `opt_onError` throws, and it should be documented what's returned in these cases. If not, the tests should be changed not to rely on this behavior.